<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>midtermClass</class>
	<widget class="QMainWindow" name="midtermClass">
		<property name="objectName">
			<string notr="true">midtermClass</string>
		</property>
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>600</width>
				<height>400</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>midterm</string>
		</property>
		<widget class="QWidget" name="centralWidget">
			<layout class="QVBoxLayout" name="verticalLayout">
				<item>
					<widget class="QLabel" name="imageLabel">
						<property name="text">
							<string>Select an image:</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QPushButton" name="selectImageButton">
						<property name="text">
							<string>Select Image</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QLabel" name="blurLabel">
						<property name="text">
							<string>Blur Radius:</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QSlider" name="blurSlider">
						<property name="orientation">
							<enum>Qt::Horizontal</enum>
						</property>
						<property name="minimum">
							<number>1</number>
						</property>
						<property name="maximum">
							<number>5</number>
						</property>
						<property name="value">
							<number>5</number>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QLabel" name="blurValueLabel">
						<property name="text">
							<string>Blur Value: 5</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QPushButton" name="applyBlurButton">
						<property name="text">
							<string>Apply Blur</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QPushButton" name="prevImageButton">
						<property name="text">
							<string>&lt;&lt; Previous</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QPushButton" name="nextImageButton">
						<property name="text">
							<string>Next &gt;&gt;</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QLabel" name="statusLabel">
						<property name="text">
							<string>Status:</string>
						</property>
					</widget>
				</item>
				<item>
					<widget class="QStatusBar" name="statusBar"/>
				</item>
			</layout>
		</widget>
		<widget class="QMenuBar" name="menuBar"/>
		<widget class="QToolBar" name="mainToolBar"/>
		<widget class="QStatusBar" name="statusBar"/>
	</widget>
	<resources/>
	<connections/>
</ui>
